src/main/java/org/broadinstitute/dsm/model/birch/TestBostonResult.java

```diff
@@ -0,0 +1,7 @@
+package org.broadinstitute.dsm.model.birch;
+
+public class TestBostonResult {
+    public String date;
+    public String sampleId;
+    public String result;
```

Comment:
This will be changed in the next push, I was testing with this json and I pushed the wrong code. But you get the idea